### PR TITLE
Update kodi.desktop fixed german writing

### PR DESCRIPTION
**Pull request by dadosch**
## Description

added the correct aplication's name in german which is different from the english one
## Motivation and Context

In Germany typically each noun begins with a capital.

## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [x] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by Jalle19** [comment]
I think the original is a mistake, the name should be just "Kodi".

**Comment by Memphiz** [comment]
Yep - agreed @Jalle19

**Comment by Jalle19** [comment]
@dadosh can you change it?

File: tools/Linux/kodi.desktop [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Version=1.0
-Name=Kodi media center
+Name=Kodi
 GenericName=Media center
```
Comment (Jalle19) [inline tools/Linux/kodi.desktop]: Could you change this to `Media Center` while you're at it?

**Comment by Jalle19** [comment]
@dadosch it seems you changed the wrong line, I meant it should be like this:

```
Name=Kodi
GenericName=Media Center
```

**Comment by dadosch** [comment]
@Jalle19 Ahh Ok. I misunderstood.

**Comment by Jalle19** [comment]
@dadosch thank you for your contribution!

**Comment by MartijnKaijser** [comment]
Commits should have been squashed into one

**Comment by Jalle19** [comment]
@MartijnKaiser they were, I used the new squash and merge feature

**Comment by Jalle19** [comment]
Stupid phone interface without nickname autocomplete...

**Comment by MartijnKaijser** [comment]
@Jalle19 right, thx. Well certainly confusing that the PR still says 4 commits :)

**Comment by Jalle19** [comment]
@MartijnKaijser indeed. I figured it's a good tool to accept contributions from people who are not that familiar with git. When you use "Squash and merge" you get the opportunity to change the commit message too, which is good.

**Pull request merged**